### PR TITLE
`wg-ffi-unwind` zulip sync

### PR DESCRIPTION
**Pull request by BatmanAoD**
#826 for `wg-ffi-unwind`; also includes #842

### CODE REVIEW

File: teams/wg-ffi-unwind.toml [hunk 1]
```diff
@@ -20,3 +26,9 @@ zulip-stream = "project-ffi-unwind"
 
 [[github]]
 orgs = ["rust-lang"]
+
+[[zulip-groups]]
+name = "WG-ffi-unwind"
+
+# Katelynn Martin has two Zulip accounts.
```
Comment (Mark-Simulacrum) [inline teams/wg-ffi-unwind.toml:33]:
```suggestion
# Katelyn Martin has two Zulip accounts.
```

Interesting. Are both of these active on an ongoing basis?

Comment (BatmanAoD) [reply teams/wg-ffi-unwind.toml]: I hadn't checked, actually, but it looks like one account was effectively unused, so I've removed that one.

**Comment by rylev** [comment]
We should make everyone who is impacted by these changes aware that they are happening. 

@CAD97, @bjorn3, @chorman0773, @cratelyn, @joshtriplett, @gnzlbg, @Amanieu

**Review by rylev — APPROVED** [review]
This looks good to me. 

As a recap, this changes the Zulip user group by adding @chorman0773. Otherwise, the Zulip group stays the same. Of course, the official membership of the working group itself does expand considerably. 

It's fine to merge this without approval from everyone impacted as reverting these changes are low-cost and trivial.

**Pull request merged**